src/views/Home.vue

```diff
@@ -0,0 +1,398 @@
+    return {
+      order: {} as any,
+      products: {} as any,
+      productIds: [] as any,
+      deliveryMethods: [
+        {
+          name: 'Store pickup',
+          value: 'STOREPICKUP'
+        },
+        {
+          name: 'Shipping',
+          value: 'STANDARD'
+        }
+      ]
+    }
+  },
+  computed: {
+    ...mapGetters({
+      deliveryMethod: 'user/getDeliveryMethod'
+    })
+  },
+  async mounted() {
+    await this.getOrder();
+  },
+  methods: {
+    async getOrder() {
+      let resp;
+      try {
+        resp = await OrderService.getOrder(this.$route.params.orderId as string);
+        if (resp.status === 200 && !hasError(resp) && resp.data) {
+          this.order = resp.data;
+          let productIds: any = new Set();
+          let isShipmentCancelled = false;
+          this.order.shipGroup.map((group: any) => {
+            group.selectedShipmentMethodTypeId = group.shipmentMethodTypeId;
+            group.items.map((item: any) => {
+              if (item.productId) productIds.add(item.productId);
+              if (item.status == 'ITEM_CANCELLED') isShipmentCancelled = true;
+            })
+            group.isShipmentCancelled = isShipmentCancelled;
+          })
+          this.productIds = [...productIds]
+          await this.fetchProducts(this.productIds)
+        } else {
+          this.order = {}
+          showToast(translate("Order not Found"))
+        }
+      } catch (error) {
+        this.order = {}
+        console.error(error)
+        showToast(translate("Something went wrong while fetching the order details"))
+      }
+    },
+
+    async fetchProducts(productIds: any) {
+      const productIdFilter = productIds.reduce((filter: string, productId: any) => {
+        if (filter !== '') filter += ' OR '
+        return filter += productId;
+      }, '');
+
+      try {
+        const resp = await ProductService.fetchProducts({
+          "filters": ['productId: (' + productIdFilter + ')'],
+          "viewSize": productIds.length
+        })
+
+        if (resp.status === 200 && !hasError(resp) && resp.data) {
+          resp.data.response.docs.forEach((product: any) => {
+            this.products[product.productId] = product
+          });
+        }
+      } catch (error) {
+        console.error(error)
+        showToast(translate("Something went wrong"))
+      }
+    },
+
+    getProduct(productId: string) {
+      return this.products[productId] ? this.products[productId] : {}
+    },
+
+    async saveShippingAddressChanges(shipGroup: any) {
+      let resp
+      const payload = {
+        "orderId": this.order.id,
+        "shipGroupSeqId": shipGroup.shipGroupSeqId,
+        "contactMechId": shipGroup.shipTo.postalAddress.id,
+        "shipmentMethod": `${this.deliveryMethod}@_NA_`,
+        "contactMechPurposeTypeId": "SHIPPING_LOCATION",
+        "facilityId": "WH",
+        "toName": `${shipGroup.editedShipmentAddress.firstName} ${shipGroup.editedShipmentAddress.lastName}`,
```

Comment:
Do not use Shipment keyword, it could be confusing. Why not just updatedAddress/editedAddress?